src/libp2p_transport_tcp.erl

```diff
@@ -68,6 +68,7 @@
          negotiated_nat=false :: boolean()
         }).
 
+-define(DEFAULT_MAX_TCP_CONNECTIONS, 1024).
```

Comment:
make this tunable, please?

Reply:
nm, ignore me, I didn't read it right.

Reply:
(just in case) 1024 is ranch's default already